common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -93,10 +93,6 @@ class WordCloudModule(WordCloudFields, XModule):
     js = {
         'js': [
             resource_string(__name__, 'js/src/javascript_loader.js'),
```

Comment:
Other things use it.  I was going to remove it, but I want to reflect that the XModule actually depends on it, rather than it be implicit (by removing the line here and getting it by virtue of other modules depending on it).